test/AST-error/bad_sample_signature.cpp

```diff
@@ -26,6 +26,8 @@ struct bad {
 // expected-note@* {{}}
 // expected-note@* {{}}
 // expected-note@* {{}}
+// expected-note@* {{}}
+// expected-note@* {{}}
```

Comment:
nit: Aren't the notes content of any interest?

Reply:
Sorry, I was just copying from above for the new diagnostic notes. Updated in [87e3a25](https://github.com/NVIDIA/cuda-quantum/pull/664/commits/87e3a25064ec7d7b8c953eb63c3563001cc47b4d).